common/web/types/src/util/common-events.ts

```diff
@@ -1,21 +1,47 @@
 import { CompilerErrorNamespace, CompilerErrorSeverity, CompilerEvent } from './compiler-interfaces.js';
 import { ErrorObject } from "ajv";
+import { constants } from '@keymanapp/ldml-keyboard-constants';
 
 const m = (code: number, message: string) : CompilerEvent => { return { code, message } };
 
 const KmcErrMask   = CompilerErrorNamespace.Types;
 // const SevInfo = CompilerErrorSeverity.Info | KmcErrMask;
 // const SevHint = CompilerErrorSeverity.Hint | KmcErrMask;
 // const SevWarn = CompilerErrorSeverity.Warn | KmcErrMask;
-// const SevError = CompilerErrorSeverity.Error | KmcErrMask;
+const SevError = CompilerErrorSeverity.Error | KmcErrMask;
 const SevFatal = CompilerErrorSeverity.Fatal | KmcErrMask;
 
 export class CommonTypesMessages {
   // structured Ajv validation error
   static Fatal_AJVError = (o:ErrorObject) => m(this.FATAL_XMLValidationError,
     `${o.instancePath}: ${o.keyword}: ${o.message} ${JSON.stringify(o.params||{})}`);
   // some other unknown Ajv error. Same as above, but without structure
-  static Fatal_UnknownXMLValidationError = (o:{errorsText: string}) => m(this.FATAL_XMLValidationError,
+  static Fatal_UnknownXMLValidationError = (o: { errorsText: string }) => m(this.FATAL_XMLValidationError,
     `Validation failed: ${o.errorsText}`);
   static FATAL_XMLValidationError = SevFatal | 0x0001;
+
+  static Error_ImportInvalidBase = (o: { base: string, path: string, subtag: string }) =>
+    m(this.ERROR_ImportInvalidBase,
+      `Import element with base ${o.base} is unsupported. Only ${constants.cldr_import_base} is supported.`);
+  static ERROR_ImportInvalidBase = SevError | 0x0001;
```

Comment:
Each identifier should be unique, XOR its severity mask. We have had messages change in severity in the past, and this keeps things sane.

Reply:
moot now that they are all errors, but good to know